sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStoreErrors.scala

```diff
@@ -145,6 +145,12 @@ object StateStoreErrors {
     new StateStoreValueSchemaNotCompatible(storedValueSchema, newValueSchema)
   }
 
+  def twsSchemaMustBeNullable(
```

Comment:
I think TWS deserves its own error collection class, but I agree this is out of scope. Let's make a follow-up.